extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -43,6 +44,17 @@
   flex: 100 !important;
   overflow: hidden !important;
   position: relative !important;
+  transition: flex-grow .5s var(--ease-out-curve) !important;
+}
+
+.i-amphtml-story-interactive-option-text-container {
+  transform: translateY(17px) !important;
+}
+
+.i-amphtml-story-interactive-post-selection
+  .i-amphtml-story-interactive-option-text-container {
+  transition: transform .5s var(--ease-out-curve) !important;
```

Comment:
Just curious: how did you pick the `.5s` duration? We've been trying to make animations pretty fast (under 300/250ms, less if the animations are smaller than the width/height of the screens) as they're mostly running on mobile.

Reply:
.5 was what I initially had in the [presentation](https://philipbell-test.web.app/demo/feedback/) but it feels slow in the context of the story. I agree that .3 and under is a good standard to keep.

The [demo link](https://poll-animation.web.app/examples/amp-story/interactives.html) has been updated with dummy data so we can get a feel of the animation as a whole.